### PR TITLE
Switch to building demo using nuget ebpf-for-windows

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Signed-off-by: Alan Jowett <alan.jowett@microsoft.com>

Resolves: #12

### CODE REVIEW

**Review by dthaler — DISMISSED** [review]

File: ebpf-for-windows-demo.sln [hunk 1]
```diff
@@ -13,10 +13,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{9171E26A
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cilium_test", "tests\cilium_tests\cilium_test.vcxproj", "{12161211-5463-4AA9-9B6E-552552B475DC}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ebpf-for-windows", "ebpf-for-windows", "{E69DC31A-4E39-4E5D-957F-FFDC31C69513}"
```
Comment (saxena-anurag) [inline ebpf-for-windows-demo.sln:16]: Since we have removed the submodule and using nuget package, we can also delete this folder completely:

`https://github.com/microsoft/ebpf-for-windows-demo/tree/main/ebpf-for-windows`

This was added to compile ebpf-for-windows submodule.

File: .gitmodules [hunk 2]
```diff
@@ -2,9 +2,6 @@
 	path = external/cilium
 	url = https://github.com/saxena-anurag/cilium.git
 	branch = anurag/cilium_xdp_bpf_code_changes
-[submodule "external/ebpf-for-windows"]
```
Comment (saxena-anurag) [inline .gitmodules:5]: We should also update the Getting Started (`https://github.com/microsoft/ebpf-for-windows-demo/blob/main/docs/GettingStarted.md`)as ebpf-for-windows is not a submodule, though we can do this in a separate PR.

**Review by saxena-anurag — APPROVED** [review]

**Review by dthaler — APPROVED** [review]
